embassy-usb-logger/src/lib.rs

```diff
@@ -67,7 +67,7 @@ impl<const N: usize> UsbLogger<N> {
         config.device_protocol = 0x01;
         config.composite_with_iads = true;
 
-        let mut builder = Builder::new(
+        let mut builder = Builder::<'d, D, 2>::new(
```

Comment:
Why hardcoding this to '2'?

Reply:
`CdcAcmClass` only requires two descriptors to initialize. I did however forget that the old default was 4, and `CdcNcmClass` requires four descriptors.